invenio_app_rdm/requests_ui/views/requests.py

```diff
@@ -209,6 +209,7 @@ def user_dashboard_request_view(request, **kwargs):
             permissions=topic["permissions"],
             is_preview=is_draft,  # preview only when draft
             is_draft=is_draft,
+            has_draft=record._record.has_draft,
```

Comment:
Wouldn't `is_draft` actually work the same? We might also need `is_published` though (see https://github.com/zenodo/zenodo-rdm/pull/1188#discussion_r2116091778 )